include/measurement_kit/internal/engine/task.hpp

```diff
@@ -0,0 +1,74 @@
+// Part of Measurement Kit <https://measurement-kit.github.io/>.
+// Measurement Kit is free software under the BSD license. See AUTHORS
+// and LICENSE for more information on the copying conditions.
+#ifndef INCLUDE_MEASUREMENT_KIT_INTERNAL_ENGINE_TASK_HPP
+#define INCLUDE_MEASUREMENT_KIT_INTERNAL_ENGINE_TASK_HPP
```

Comment:
This code has been extracted verbatim from `src/libmeasurement_kit/engine/task.hpp`.